cmd/kyverno/main.go

```diff
@@ -92,6 +93,8 @@ func main() {
 	flag.IntVar(&clientRateLimitBurst, "clientRateLimitBurst", 0, "Configure the maximum burst for throttle. Uses the client default if zero.")
 	flag.Func(toggle.AutogenInternalsFlagName, toggle.AutogenInternalsDescription, toggle.AutogenInternalsFlag)
 	flag.DurationVar(&webhookRegistrationTimeout, "webhookRegistrationTimeout", 120*time.Second, "Timeout for webhook registration, e.g., 30s, 1m, 5m.")
+	flag.IntVar(&changeRequestLimit, "changeRequestLimit", 1000, "Configure the maximum count for report change requests.")
```

Comment:
This is per namespace, right? Maybe state as: `Configure the maximum allowed report change requests per namespace`

Reply:
It is based on the change request types:

reportchangerequest -> per namespace
clusterreportchangerequest -> cluster-wide

Reply:
I see, Seems like `reportchangerequest` is still common across, and more descriptive than `changeRequest`. 

```golang
flag.IntVar(&changeRequestLimit, "maxReportChangeRequests", 1000, "maximum pending report change requests per namespace or for the cluster-wide policy report")

```